README.md

```diff
@@ -12,23 +12,23 @@
 
 # Easy Deploy - Chrome Extension
 
-This is a chrome extension... WIP
+This is a chrome extension that allows you to manually trigger a GitHub Action.
 
 ### How to run locally
 
-1. Clone repo and install dependencies
-2. Run the server `npm start` _(watch mode enabled)_
+1. Clone this repository and install dependencies using `npm install` inside the cloned folder
+2. Run the local development server using `npm start` _(watch mode enabled)_
 3. Open a new chrome tab with the following URL: <chrome://extensions/>
-4. Enable developer mode and click LOAD UNPACKED
+4. Enable developer mode and click **LOAD UNPACKED**
 
   ![Screenshot from documentation](https://developer.chrome.com/static/images/get_started/load_extension.png "Screenshot from documentation")
   
   [Chrome documentation and image source](https://developer.chrome.com/extensions/getstarted#manifest)
 
-5. Select the dist folder
-6. Go to GitHub page, refresh and click on the extension to see the popup working
+5. Select the `dist/` folder
+6. Open GitHub.com page and navigate to your repository, click on the extension to see the popup working
 
-Note: Keep in mind when developing in local, either background.ts & contentScript.ts files are being updated dinamically
+Note: Keep in mind when developing locally, neither `background.ts` nor `contentScript.ts` are being updated dynamically. You have to manually update the extension content from the Chrome extension's page.
```

Comment:
actually both files are dynamically updated thanks to `webpack-extension-reloader` - going to update that little part in the next readme update :)